snap/validate.go

```diff
@@ -394,13 +394,34 @@ func validateAppSocket(socket *SocketInfo) error {
 	return validateSocketAddr(socket, "listen-stream", socket.ListenStream)
 }
 
+func injectFakeExternalApps(apps map[string]*AppInfo) {
```

Comment:
I think this warrants a comment. On the function name alone I would expect to see this in a testing file.